### ISSUE TITLE
Cleaning house

### ISSUE THREAD

**Issue by nidzieko** [Horn-Point-Labratory/moos-ivp-hpl#4]
We need to add comments and/or a directory structure to mark whether something is: 
1) a duplicate of a process from someone else's repo  
2) a modification of someone else's code
3) working or in development

**Comment by sgillen** [comment]
I agree with giving credit where credit is due, we could consider having a separate directory for third party apps, but that will complicate the build process. I think it's probably enough to give credit in the comment header, the readme, and the description on GitHub. 
I believe you can make a repo private for free if we host it on one of our individual accounts. I think a good idea is to have a clean public branch on the hpl page, and have a private testing branch on one of our own pages. When we finish polishing a feature we can push it to the public repo. Not quite sure how to do this, but one of the features git seems to advertise is that it makes this easy. So I'll look into setting something like that up.

-Sean Gillen
